apps/right-to-rent-check/acceptance/features/tenancy-start.js

```diff
@@ -6,31 +6,36 @@ const steps = require('../../');
 Feature('Tenancy Start Page');
 
 Before((
-  I,
-  tenancyStartPage
+  I
 ) => {
-  I.visitPage(tenancyStartPage, steps);
+  I.amOnPage('/');
```

Comment:
You should use `visitPage`. It does `amOnPage` and more https://github.com/UKHomeOffice/so-acceptance/blob/master/steps.js#L17-L30